### ISSUE TITLE
On Reboot cacerts\": x509: certificate has expired or is not yet valid: cur> lines 1-45...skipping..

### ISSUE THREAD

**Issue by MayanNetwork** [k3s-io/k3s#6368]
**Environmental Info:**

K3s Version: 
k3s version v1.25.0+k3s1 (26e94057)

Node(s) CPU architecture, OS, and Version: 

Linux Mayan 4.18.0-305.rt7.72.el8.x86_64 #1 SMP PREEMPT_RT Thu Apr 29 12:30:00 EDT 2021 x86_64 x86_64 x86_64 GNU/Linux

Cluster Configuration:

Airgap Environment, single node cluster, one server

**Describe the bug:**
Upon rebot K3S fails to run, indicating invalid certificate.

Jun 22 11:12:01 Mayan k3s[1600]: time="2018-06-22T11:12:01Z" level=error msg="CA cert validation failed: Get \"https://127.0.0.1:6443/cacerts\": x509: certificate has expired or is not yet valid: cur>
lines 1-45...skipping...
...
...
Jun 22 11:12:01 Mayan k3s[1600]: time="2018-06-22T11:12:01Z" level=error msg="CA cert validation failed: Get \"https://127.0.0.1:6443/cacerts\": x509: certificate has expired or is not yet valid: cur>
Jun 22 11:12:01 Mayan k3s[1600]: I0622 11:12:01.963359    1600 instance.go:574] API group "internal.apiserver.k8s.io" is not enabled, skipping.
Jun 22 11:12:02 Mayan k3s[1600]: W0622 11:12:02.119655    1600 genericapiserver.go:656] Skipping API authentication.k8s.io/v1beta1 because it has no resources.

**Steps To Reproduce:**
Followed the airgap install.  Needed a local repo to install selinux-container.  disabled firewalld and selinux.
K3S installs and runs (even created a pod and ran an image from a podman registry)

Reboot, results in the error.

But systemctl stop and restart fixes it.

A reboot repeats the error

**Expected behavior:**

K3S autoboots happy

**Additional context / logs:**

NAME   STATUS   ROLES                  AGE   VERSION
Mayan   Ready    control-plane,master   67m   v1.25.0+k3s1

[journal.txt.txt](https://github.com/k3s-io/k3s/files/9880761/journal.txt.txt)

[system_status_k3s.txt](https://github.com/k3s-io/k3s/files/9880755/system_status_k3s.txt)

**Comment by brandond** [comment]
> 2018-06-22T11:12:01Z is before 2018-06-22T11:13:29Z

Unless you're using a time machine to post this issue, none of these times are correct. Please set your system time correctly, and make sure that the system time when starting K3s is not set to a time before the cluster was initially created.

**Comment by MayanNetwork** [comment]
Yeah, it is an embedded board in an airgap subnet.   I changed the date time but on reboot it goes back to June 2018.  

I was thinking of temporarily changing the time to before the one in the error and create a certificate, but I have feeling there are others too.

**Comment by brandond** [comment]
That's going to be a problem. If you can't get a functional RTC with a battery, you'll need to at least manually script something to ensure that time consistently flows forward, instead of resetting or going backwards, on this node.
